tests/test_packaging.py

```diff
@@ -173,7 +169,8 @@ def test_pack_in_pex(pyarrow_version, expectation):
     if sys.version_info.minor in {8, 9} and pyarrow_version == "0.13.0":
         return
     with tempfile.TemporaryDirectory() as tempdir:
-        requirements = ["protobuf==3.19.6", "tensorflow==2.5.2", f"pyarrow=={pyarrow_version}"]
+        requirements = ["tensorflow", f"pyarrow=={pyarrow_version}",
+                        "urllib3<2.0"]

```

Comment:
why do we need this constraint on urllib3 ?

Reply:
needed with the version change of T for the tests to pass